test/functional/test_runner.py

```diff
@@ -139,6 +139,8 @@ class UnicodeOnWindowsError(ValueError):
     'wallet_data_deposit.py',
     'wallet_submit_tx.py',
     'wallet_select_utxos.py',
+    'wallet_sweep.py',
```

Comment:
Please rename the file to `wallet_sweep_address.py` since this is only about sweeping addresses, to avoid ambiguity.